include/nbl/system/CApplicationFrameworkAndroid.h

```diff
@@ -0,0 +1,158 @@
+#ifndef	_NBL_SYSTEM_C_APPLICATION_FRAMEWORK_ANDROID_H_INCLUDED_
+#define	_NBL_SYSTEM_C_APPLICATION_FRAMEWORK_ANDROID_H_INCLUDED_
+#ifdef _NBL_PLATFORM_ANDROID_
+#include "nbl/core/declarations.h"
+#include <android_native_app_glue.h>
+#include <android/sensor.h>
+#include <android/log.h>
+namespace nbl::system
+{
+
+    class CApplicationFrameworkAndroid : public core::IReferenceCounted
+    {
+    public:
+        void onStateSaved(android_app* params)
+        {
+            return onStateSaved_impl(params);
+        }
+        void onWindowInitialized(android_app* params)
+        {
+            return onWindowInitialized_impl(params);
+        }
+        void onWindowTerminated(android_app* params)
+        {
+            return onWindowTerminated_impl(params);
+        }
+        void onFocusGained(android_app* params)
+        {
+            return onFocusGained_impl(params);
+        }
+        void onFocusLost(android_app* params)
+        {
+            return onFocusLost_impl(params);
+        }
+        virtual void draw(android_app* params) = 0;
```

Comment:
dont call it `draw`, maybe `loop`, `work` or `workLoop` (or `[work]LoopBody`) 

I'll let @Crisspl decide.